pkg/tools/opentelemetry/exporter.go

```diff
@@ -73,5 +75,22 @@ func InitMetricExporter(ctx context.Context, exporterURL string) *sdkmetric.Expo
 		return nil
 	}
 
-	return &exporter
+	return sdkmetric.NewPeriodicReader(
+		exporter,
+		sdkmetric.WithInterval(10*time.Second),
```

Comment:
Isn't it better to make it configurable?

Reply:
do you mean the 10 seconds?